Userland/Applications/DisplaySettings/MonitorWidget.cpp

```diff
@@ -146,6 +146,15 @@ void MonitorWidget::redraw_desktop_if_needed()
         painter.draw_tiled_bitmap(m_desktop_bitmap->rect(), *scaled_bitmap);
     } else if (m_desktop_wallpaper_mode == "Stretch"sv) {
         painter.draw_scaled_bitmap(m_desktop_bitmap->rect(), *m_wallpaper_bitmap, m_wallpaper_bitmap->rect(), 1.f, Gfx::ScalingMode::BilinearBlend);
+    } else if (m_desktop_wallpaper_mode == "Fill"sv) {
+        auto wallpaper_rect = m_wallpaper_bitmap->rect();
+        auto scale_x = static_cast<float>(m_desktop_bitmap->width()) / wallpaper_rect.width();
```

Comment:
This is the same as `sw` which already exists, right?

Maybe this does the same in a bit shorter:

```
auto new_size = (wallpaper_rect.size().to_type<float>() * max(sw, sh)).to_type<int>();
auto stretched_rect = Gfx::IntRect({}, new_size).centered_within(m_desktop_bitmap->rect());
```

? (not tested)